### ISSUE TITLE
Zelda checks show up when Skip Child Zelda is selected

### ISSUE THREAD

**Issue by aaarizpe** [StripesOO7/oot-tracker#12]
These two checks in Hyrule Castle show up even though I'm playing with shuffle_child_trade set to Skip Child Zelda, which makes it so that you start the game with Zelda's Letter and the Song from Impa, and there is nothing at those checks.

![image](https://github.com/user-attachments/assets/4d8bb806-9b37-4b71-9f0a-fc975c91b3dc)

**Comment by StripesOO7** [comment]
fixed.
are there any more checks the are not present when skipping child trade quest?

**Comment by aaarizpe** [comment]
Thanks, the fix worked. I think the only other check that's skipped is Malon's Egg in Hyrule Castle, but that one was already working correctly.

BTW, thanks for quick fixes, and for making this tracker - I just started playing with it and it's great! Through experimenting with the latest .zip, I found a few more checks that don't show up as reachable even with all items - I'm going to go write those up now.
